suse/sle-hpc-15.x/sle-hpc-15-sp4/install_utils.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+set -ex
+
+# Install pre-reqs and development tools
+#
+
+# Add additional repositories
+
+#-------------------------------------------------------------------
+# Intel provides oneapi RPM packages for SUSE, so we only need to add the repositories and install the packages
+#-------------------------------------------------------------------
+# see
+# https://www.intel.com/content/www/us/en/develop/documentation/installation-guide-for-intel-oneapi-toolkits-linux/top/installation/install-using-package-managers/yum-dnf-zypper.html
+
+# import package signing keys
+rpm --import https://yum.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+# add repository
+zypper -n addrepo -f -g https://yum.repos.intel.com/oneapi oneAPI
+# fetch key
+zypper --non-interactive --gpg-auto-import-keys refresh
+
+# list all packages
+# sudo -E zypper pa -ir oneAPI
+#-------------------------------------------------------------------
+
+#-------------------------------------------------------------------
+# Nvidia provide certified packages for SLES 15, so we only need to add the repositories and install the packages
+#-------------------------------------------------------------------
+# import cuda signing key
+rpm --import https://developer.download.nvidia.com/compute/cuda/repos/sles15/x86_64/D42D0685.pub
+
+# CUDA driver (nvidia provides a repo file)
+zypper addrepo -f -g https://developer.download.nvidia.com/compute/cuda/repos/sles15/x86_64/cuda-sles15.repo
+# fetch key
+zypper --non-interactive --gpg-auto-import-keys refresh
+
+#-------------------------------------------------------------------
+# Container Repository
+#-------------------------------------------------------------------
+# Docker is shipped with SLES by default
+# with SLES HPC we need to enable the Container repository
+SUSEConnect -p sle-module-containers/15.4/x86_64
+
+#-------------------------------------------------------------------
+# nvidia container repo
+#-------------------------------------------------------------------
+# see https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html
+# Check https://nvidia.github.io/libnvidia-container
+zypper addrepo -f -g https://nvidia.github.io/libnvidia-container/sles15.3/libnvidia-container.repo
+# fetch key
+zypper --non-interactive --gpg-auto-import-keys refresh
+
+#-------------------------------------------------------------------
+# Add SUSE Package Hub
+# byacc is only in packagehub
+SUSEConnect -p PackageHub/15.4/x86_64
+#-------------------------------------------------------------------
+
+#
+## SLES HPC ship with many HPC packages already, so no need to build it - simple install is enough
+#
+# as the default cloud image does only install parts of the HPC toolchain we use the pattern
+zypper install -y --type pattern hpc_libraries
```

Comment:
I believe this should be a comment

Reply:
no, what we have available in the product and what the SLE HPC images has pre-installed within the marketpalce image is different. The marketplace image is has only a minimal set of packages installed (just enough os principle).
So using a pattern (= a group/list of packages) to install our default setup is needed